### PR TITLE
Fix OpenShift Keycloak + Sql Server scenarios

### PR DESCRIPTION
**Pull request by Sgitario**
The problem is that in RH SSO 7.3 is expected `Log in to test-realm` and in RH SSO 7.4 is `Sign in to test-realm`.

I fixed it by using a regular expression.

Fix https://github.com/quarkus-qe/quarkus-test-suite/issues/259

### CODE REVIEW

**Comment by Sgitario** [comment]
run tests

**Comment by Sgitario** [comment]
run tests

File: sql-db/sql-app/src/test/resources/test.properties [hunk 1]
```diff
@@ -1,6 +1,3 @@
 ts.app.log.enable=true
 ts.database.log.enable=true
-ts.database.openshift.use-internal-service-as-url=true
-ts.mssql.log.enable=true
-ts.mssql.openshift.template=/mssql.yaml
```
Comment (jsmrcka) [inline sql-db/sql-app/src/test/resources/test.properties:5]: Is dropping this custom template fixing anything, or has it just become obsolete?

**Review by jsmrcka — APPROVED** [review]

**Comment by jsmrcka** [comment]
JVM failures are caused by missing RH registry configuration. Not related to the PR.

File: sql-db/sql-app/src/test/resources/test.properties (hunk 1 above)
Comment (Sgitario) [reply sql-db/sql-app/src/test/resources/test.properties:5]: Yes, it's fixing the OpenShift Mssql scenario. This scenario without this configuration works fine.

Comment (jsmrcka) [reply sql-db/sql-app/src/test/resources/test.properties:5]: I see. I didn't notice the scenario failing. Thanks for the explanation.

**Pull request merged**